Pass Google logs to Symfony logger

colinfrei opened the pull request.

Nyholm commented:
Thank you for this PR. This is a great idea. But what if the project got not logger defined? 
You should make the logger an optional parameter and make sure to check for null before you create the `$googleLogger`.

Use: 
```yaml
arguments: [%happy_r_google_api%, @?logger]
```

colinfrei commented:
I wasn't aware that people remove the Logger :)
Adjusted to make the logger optional. I also added a monolog channel tag, to make it easier to filter these messages.

Nyholm commented:
Cool. Thank you for this!
